src/main/java/seedu/address/logic/commands/editcommand/EditMentorCommand.java

```diff
@@ -2,18 +2,32 @@
 
```

Comment:
Also not sure if you already have it, but for the EditCommand class can you include a MESSAGE_USAGE as a string to be shown in case the user inputs an incorrect command. It's in the og AB3 edit command.

Reply:
Same for the list command.

Reply:
Will do